fast64_internal/f3d/f3d_gbi.py

```diff
@@ -5176,24 +5419,51 @@ class DPFullSync(GbiMacro):
     def to_binary(self, f3d, segments):
         return gsDPNoParam(f3d.G_RDPFULLSYNC)
 
+    def bleed(self, LastGfxList: GfxList, CurList: GfxList, curIndex: int):
+        return False
+
 
 @dataclass
 class DPTileSync(GbiMacro):
     def to_binary(self, f3d, segments):
         return gsDPNoParam(f3d.G_RDPTILESYNC)
 
+    # will be bled if there are two of these syncs, at most only one pipesync
+    # is ever required after rendering triangles, and subsequent cmds rdp attr changes
+    def bleed(self, LastGfxList: GfxList, CurList: GfxList, curIndex: int):
```

Comment:
For these sync commands, it seems like these bleed functions always return True?

Reply:
Ok this is a silly bug lol. It should be `return False` on the outside if the loop.